### PR TITLE
[RHELC-854] Man/help page to mention config file location

### PR DESCRIPTION
**Pull request by hosekadam**
Small improvement of docustring of `--config-file` option.

Jira Issue: [[RHELC-854](https://issues.redhat.com/browse/RHELC-854)]

Checklist
- [x] PR meets acceptance criteria specified in the Jira issue
- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [ ] Code and tests are documented properly
- [x] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending`

### CODE REVIEW

File: convert2rhel/toolopts.py [hunk 1]
```diff
@@ -202,7 +202,11 @@ def _register_options(self):
             help="A configuration file to safely provide either a user password or an activation key for registering"
             " the system through subscription-manager. Alternatively, passing these values through the"
             " --activationkey or --password option would leak them through a list of running processes."
-            " Example of this file in /etc/convert2rhel.ini",
+            " Possible locations of the file and it's priority:"
+            " 1) user specified and passed by -c, --config-file option; highest priority"
+            " 2) ~/.convert2rhel.ini; lower priority"
+            " 3) /etc/convert2rhel.ini; the lowest priority"
+            " Example of the file in /etc/convert2rhel.ini",
```
Comment (SpyTec) [inline convert2rhel/toolopts.py:209]: I think we should do in the opposite order kind of. Mention that `/etc/convert2rhel.ini` file exists and should be used, but that they can overwrite it by using either `-c|--config-file` or `~/.convert2rhel.ini`

Comment (hosekadam) [reply convert2rhel/toolopts.py:209]: @SpyTec Do you mean something like:
```
The example of the file is in /etc/convert2rhel.ini. This file can
be used in this path as the low priority one or in  ~/.convert2rhel.ini
with highest priority. Also is possible to specify different path using 
this option which has the highest priority.
```

Comment (abadger) [reply convert2rhel/toolopts.py:209]: I think Eric basically just wants the list reversed.  Something like:
```
            " Possible locations of the file and it's priority:"
            " 1) /etc/convert2rhel.ini; systemwide, default values"
            " 2) ~/.convert2rhel.ini; overrides any system defaults with user supplied values"
            " 3) user specified and passed by -c, --config-file option; highest priority, options in this file override anything in (1) and (2) "
```

File: convert2rhel/toolopts.py [hunk 2]
```diff
@@ -202,7 +202,11 @@ def _register_options(self):
             help="A configuration file to safely provide either a user password or an activation key for registering"
             " the system through subscription-manager. Alternatively, passing these values through the"
```
Comment (mportman12) [inline convert2rhel/toolopts.py:203]: The sentence "Alternatively..." is somewhat confusing. Recommend rewriting as:
"This is more secure than passing these values through the --activationkey or --password option, which might leak the values through a list of running processes."

File: convert2rhel/toolopts.py (hunk 1 above)
Comment (hosekadam) [reply convert2rhel/toolopts.py]: Ok, I understand. I've updated the message to your suggestion and changed the sentence alternatively to Miriam's suggestion.

@SpyTec You can re-review that if it's fine

Comment (SpyTec) [reply convert2rhel/toolopts.py]: Problem here is though that if I was a new user to convert2rhel reading this I'd assume I have to provide the config file always. So I'd provide `/etc/convert2rhel.ini` even though we already load from there. This should be rewritten a bit to say that the file provided here overwrites the files in etc and user dir and also explain that user dir has higher priority than etc one

Comment (hosekadam) [reply convert2rhel/toolopts.py]: OK, I'll try to improve that and than consult it with Miriam.

Comment (SpyTec) [reply convert2rhel/toolopts.py]: @hosekadam consult with whomever is easier or faster for you

File: convert2rhel/toolopts.py [hunk 3]
```diff
@@ -199,10 +199,14 @@ def _register_options(self):
...
-            help="A configuration file to safely provide either a user password or an activation key for registering"
-            " the system through subscription-manager. Alternatively, passing these values through the"
-            " --activationkey or --password option would leak them through a list of running processes."
-            " Example of this file in /etc/convert2rhel.ini",
+            help="The configuration file is an optional way to safely pass either a user password or an activation key"
+            " to the subscription-manager to register the system. This is more secure than passing these values"
+            " through the --activationkey or --password option, which might leak the values"
+            " through a list of running processes."
+            " You can edit the pre-installed configuration file template at /etc/convert2rhel.ini or create a new"
+            " configuration file at ~/.convert2rhel.ini. The convert2rhel utility loads the configuration from either"
```
Comment (r0x0d) [inline convert2rhel/toolopts.py:207]: Not sure if we want to actually write instead:

```suggestion
            " configuration file at ~/.convert2rhel.ini. The Convert2RHEL utility loads the configuration from either"
```

**Review by r0x0d — APPROVED** [review]
Just one small comment, but otherwise, I think it's all okay. The wording there sounds clear and very well understanding from a user perspective.

File: convert2rhel/toolopts.py (hunk 3 above)
Comment (SpyTec) [reply convert2rhel/toolopts.py:207]: That is for the product offering, we use all lowercase for the tool itself

**Pull request merged**